app/shared/ui-wizard/src/commonMain/kotlin/ui/wizard/step/SelectTheme.kt

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (C) 2024-2025 OpenAni and contributors.
+ *
+ * 此源代码的使用受 GNU AFFERO GENERAL PUBLIC LICENSE version 3 许可证的约束, 可以在以下链接找到该许可证.
+ * Use of this source code is governed by the GNU AGPLv3 license, which can be found at the following link.
+ *
+ * https://github.com/open-ani/ani/blob/main/LICENSE
+ */
+
+package me.him188.ani.app.ui.wizard.step
+
+import androidx.compose.foundation.LocalIndication
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.horizontalScroll
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.FlowRow
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.selection.selectable
+import androidx.compose.foundation.selection.selectableGroup
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.RadioButton
+import androidx.compose.material3.Switch
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.unit.dp
+import me.him188.ani.app.data.models.preference.DarkMode
+import me.him188.ani.app.data.models.preference.ThemeSettings
+import me.him188.ani.app.ui.foundation.LocalPlatform
+import me.him188.ani.app.ui.foundation.text.ProvideContentColor
+import me.him188.ani.app.ui.foundation.theme.AniThemeDefaults
+import me.him188.ani.app.ui.foundation.theme.appColorScheme
+import me.him188.ani.app.ui.settings.SettingsTab
+import me.him188.ani.app.ui.settings.framework.components.TextItem
+import me.him188.ani.app.ui.settings.tabs.theme.ColorButton
+import me.him188.ani.app.ui.settings.tabs.theme.DiagonalMixedThemePreviewPanel
+import me.him188.ani.app.ui.settings.tabs.theme.ThemePreviewPanel
+import me.him188.ani.app.ui.theme.themeColorOptions
+import me.him188.ani.app.ui.wizard.WizardLayoutParams
+import me.him188.ani.utils.platform.isAndroid
+
+@Composable
+private fun renderThemeModeText(mode: DarkMode): String {
+    return when (mode) {
+        DarkMode.LIGHT -> "亮色"
+        DarkMode.DARK -> "暗色"
+        DarkMode.AUTO -> "自动"
+    }
+}
+
+@Composable
+internal fun SelectTheme(
+    config: ThemeSettings,
+    onUpdate: (ThemeSettings) -> Unit,
```

Comment:
做一个 ui 的 ThemeSettings 吧, 数据层的 `ThemeSettings` 这个里面有不少 deprecation. 

```kotlin
data class ThemeSelectionState(
    val darkMode,
    val useDyn,
    val color,
)
```

```suggestion
    state: ThemeSelectionState,
    onDarkThemeChange: (DarkMode) -> Unit,
    onDynamicThemeChange: () -> Unit,
```